readme.md

```diff
@@ -8,7 +8,7 @@ Designed as a partner tool to https://github.com/buildkite/ecs-run-task and shar
 
 install it
 ```bash
-go get -u github.com/99designs/ecs-upload-task
+go install -u github.com/99designs/ecs-upload-task
```

Comment:
I don't think the `-u` is valid for `go install`, and from memory you may also need to specify version?
`go install github.com/99designs/ecs-upload-task@latest`

Reply:
Thanks. This was clearly me just thinking I could switch get and install. Updated